pages/index.tsx

```diff
@@ -289,43 +304,39 @@ export default function Home({ featuredStorefronts }: HomeProps) {
     onLoadMore: loadMoreListings,
   });
 
+  function getListingsToDisplay(allListings: Listing[]): Listing[] {
+    //@ts-ignore
+    return compose(
+      // @ts-ignore
+       filter(filters[filterBy]),
+       // @ts-ignore
+       sortWith(sorts[sortBy])
+       // @ts-ignore
+     )(allListings)
+   }
```

Comment:
```suggestion
  const applyLIstingFilterAndSort = 
    //@ts-ignore
compose(
      // @ts-ignore
       filter(filters[filterBy]),
       // @ts-ignore
       sortWith(sorts[sortBy])
       // @ts-ignore
     )
  
```

No need to re wrap the call to compose in another function. You can give it a friendly name by setting the compose to a const.